controllers/rollingupgrade_controller.go

```diff
@@ -244,28 +248,133 @@ func (r *RollingUpgradeReconciler) CallKubectlDrain(ctx context.Context, nodeNam
 	errChan <- nil
 }
 
-func (r *RollingUpgradeReconciler) WaitForTermination(nodeName string, nodeInterface v1.NodeInterface) (bool, error) {
+func (r *RollingUpgradeReconciler) WaitForDesiredInstances(ruObj *upgrademgrv1alpha1.RollingUpgrade) error {
+	var (
+		started = time.Now()
+	)
+	for {
```

Comment:
Can we use inverse-exponential-backoff here?

Reply:
Yes, great idea